blocks/ai-text/package.json

```diff
@@ -54,7 +54,7 @@
     "blockType": {
       "entryPoint": "react"
     },
-    "name": "@hash/ai-text",
+    "name": "@luisbett/ai-text",
```

Comment:
I think this diff was unintentional, right?
```suggestion
    "name": "@hash/ai-text",
```

Reply:
Yup, my bad!